MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Tomography/TomographyIfaceModel.h

```diff
@@ -250,7 +253,7 @@ class MANTIDQT_CUSTOMINTERFACES_DLL TomographyIfaceModel {
   std::string buildOutReconstructionDir(const std::string &samplesDir,
                                         bool) const;
 
-  bool processIsRunning(int pid);
+  bool processIsRunning(qint64 pid);
```

Comment:
Does this modify the process or just return a status - it could be `const` to enforce the fact it doesn't accidentally change the process